MANIFEST.in

```diff
@@ -1,2 +1,2 @@
-include *.rst LICENSE Makefile
+include *.rst Makefile
```

Comment:
Probably the same as we use for Kinto. https://github.com/Kinto/kinto/blob/master/LICENSE
Apache 2